distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -573,7 +448,7 @@ get_source_location_builtin frames_to_skip = @Builtin_Method "Meta.get_source_lo
    Arguments:
    - value: The value for which to display the type.
 get_simple_type_name : Any -> Text
-get_simple_type_name value = @Builtin_Method "Meta.get_simple_type_name"
+get_simple_type_name value = @Tail_Call Meta_Helpers.get_simple_type_name_builtin value
```

Comment:
Why the `@Tail_Call`?

Reply:
- the call is tail call
- I don't want to increase the stack size
- especially in `get_source_location` with its `skip_frames` it matters
- I admit I have no measurement whether using `@Tail_Call` in these cases makes things better or worse